### DISCUSSION TITLE
My profile tab is not working

### DISCUSSION THREAD

**Discussion by patrickstar022** [community/community#133206] [Mobile] [answered]
### Select Topic Area

Question

### Body

![Screenshot_2024-07-23-12-07-28-741_com github android](https://github.com/user-attachments/assets/44a22140-b584-4dd6-960a-e376e8bb0dae)

**Comment by stevepopovich** [comment, marked as answer]
Hi @ParagSenDeka sorry this is happening. Could you provide more information? Are other tabs loading? What version of the app are you on? You can see the version in the bottom of the settings tab

**Reply by patrickstar022** [reply]
Yes other tabs loading, and me using latest version of the app

**Reply by patrickstar022** [reply]
GitHub Mobile v1.169.1 (799).

**Reply by stevepopovich** [reply]
Have you tried reinstalling the app? Not an ideal solution, but could fix

**Reply by patrickstar022** [reply]
Reinstalled it, It worked. thank you steve
